services/distribution/src/main/java/app/coronawarn/server/services/distribution/assembly/io/IO.java

```diff
@@ -63,8 +62,7 @@ public static void writeBytesToFile(byte[] bytes, File outputFile) {
     try (FileOutputStream outputFileStream = new FileOutputStream(outputFile)) {
       outputFileStream.write(bytes);
     } catch (IOException e) {
-      logger.error("Could not write file {}", outputFile);
-      throw new RuntimeException(e);
+      throw new RuntimeException("Could not write file " + outputFile, e);
```

Comment:
We propose the less generic `UncheckedIOException` which extends `RuntimeException` and also wraps an `IOException`

Reply:
Replaced with UncheckedIOException.